tests/unit/combine_results/expected_output.md5

```diff
@@ -1 +1,2 @@
 1529b2a9ad9485a578805fe4ca8b0c11  ./tests/unit/combine_results/combined_MotEvo_results.tsv
+f54d22e3d35b37f5cfe476414ccc09c1  ./tests/unit/ProbabilityvsSequences.pdf
```

Comment:
GitHub says this file is missing a newline at the end.

Reply:
Also - why is this file loosely hanging under a directory `unit` for unit tests? it should not be like that - it should be under a dedicated unit test for the heatmap-plotting rule. Please keep the repository clean.